src/components/ReactPlayer/ReactPlayer.tsx

```diff
@@ -308,38 +322,34 @@ export const ReactPlayerBlock = React.forwardRef<ReactPlayerBlockHandler, ReactP
             setEnded(true);
         }, [loop, playerRef]);
 
-        const renderCustomBarControls = useCallback(
-            (isMuted: boolean, elapsedTimePercent: number) => {
-                if (controls !== MediaVideoControlsType.Custom || !isPlaying) {
-                    return null;
-                }
+        const onPlayClick = useCallback(() => {
+            if (isPlaying) {
+                onPause();
+            } else {
+                onPlay();
+            }
+        }, [isPlaying, onPlay, onPause]);
 
-                return (
-                    <CustomBarControls
-                        className={b(
-                            'custom-bar-controls',
-                            {muted: isMuted},
-                            customBarControlsClassName,
-                        )}
-                        mute={{
-                            isMuted,
-                            changeMute: (event: React.MouseEvent) => {
-                                event.stopPropagation();
-                                changeMute(isMuted);
-                            },
-                        }}
-                        elapsedTimePercent={elapsedTimePercent}
-                    />
-                );
-            },
-            [controls, isPlaying, customBarControlsClassName, changeMute],
-        );
+        const handleClick = useCallback(() => {
+            if (customControlsType === CustomControlsType.WithMuteButton) {
+                changeMute(muted);
+            } else {
+                onPlayClick();
+            }
+        }, [changeMute, customControlsType, muted, onPlayClick]);
+
+        const onFocusIn = useCallback(() => setHovered(true), []);
+        const onFocusOut = useCallback(() => setHovered(false), []);
 
         return (
             <div
-                className={b({wrapper: !currentHeight}, className)}
+                className={b({wrapper: !currentHeight, controls, started, hovered}, className)}
```

Comment:
Yes, initially it was made by css only, but one should render shadow over player also when play/pause button has focus. Because, when user focuses to play/pause button, the outline with white color renders, but the video itself may be also white. Thus, the outline becomes invisible